examples/todos/common/src/test/kotlin/org/reduxkotlin/examples/todos/TodosReducerSpec.kt

```diff
@@ -0,0 +1,111 @@
+package org.reduxkotlin.examples.todos
+
+import ch.tutteli.atrium.api.cc.en_GB.toBe
+import ch.tutteli.atrium.verbs.expect
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+object TodosReducerSpec : Spek({
+    describe("todos reducer") {
+        it("should handle AddToDo") {
+            expect(
+                todosReducer(
```

Comment:
Nitpick, but I think this could be formatted as

```
todosReducer(
    listOf(), 
    AddTodo(text = "Run the tests")
)
```

Reply:
I think all the `AddTodo`s can be on one line

Reply:
done